.gitignore

```diff
@@ -50,5 +50,4 @@ cmd/simulator/.simulator/*
 # goreleaser
 dist/
 
-# generator rpc file for e2e tests
-contract-examples/dynamic_rpc.json
\ No newline at end of file
+node_modules
```

Comment:
hm this is already specified under `contract-examples/.gitignore`. did we specifically need to ignore this in root directory?